packages/lesswrong/lib/collections/users/helpers.tsx

```diff
@@ -446,3 +452,13 @@ export const getAuth0Id = (user: DbUser) => {
   }
   throw new Error("User does not have an Auth0 user ID");
 }
+/**Finds a user matching on email, searches case INsensitively.
+ * Currently searches both email and emailS fields, though the later should ideally be full deprecated
+ */
+export const userFindOneByEmail = async function(email: string): Promise<DbUser|null> {
+  return await mongoFindOne("Users", {$or: [{email}, {['emails.address']: email}]}, {collation: {locale: 'en', strength: 2}})
+};
+
+export const usersFindAllByEmail: (email: string) => Promise<Array<DbUser|null>> = async function(email: string)  {
+  return await mongoFind("Users", {$or: [{email}, {['emails.address']: email}]}, {collation: {locale: 'en', strength: 2}})
+};
```

Comment:
I don't love the use of `mongoFindOne` and `mongoFind`; is there any upside to using them over just using the collection `find` and `findOne` methods directly?  Seems like an unnecessary layer of indirection